lib/associations/belongs-to-many.js

```diff
@@ -464,6 +464,10 @@ BelongsToMany.prototype.injectGetter = function(obj) {
       }
     }
 
+    if(options.hasOwnProperty('schema')){
+      model = model.schema(options.schema,options.schemaDelimiter || '');
```

Comment:
Note on styling, we generally have a space between `if` and `(`.
We generally also have a space after a comma in a options set, so `options.schema, options.schemaDelimiter`:

This applies to the rest of the code aswell.